client/jetpack-onboarding/steps/contact-form.jsx

```diff
@@ -29,9 +30,20 @@ class JetpackOnboardingContactFormStep extends React.PureComponent {
 	};
 
 	render() {
-		const { getForwardUrl, translate } = this.props;
+		const { getForwardUrl, settings, translate } = this.props;
 		const headerText = translate( "Let's shape your new site." );
 		const subHeaderText = translate( 'Would you like to get started with a Contact Us page?' );
+		const hasContactForm = !! get( settings, 'addContactForm' );
+		const tileProps = hasContactForm
+			? {
+					description: translate( 'Your contact form has been created.' ),
+				}
+			: {
+					buttonLabel: translate( 'Add a contact form' ),
+					description: translate(
+						'Not sure? You can skip this step and add a contact form later.'
+					),
+				};
```

Comment:
Maybe consider doing it separately for each prop (`buttonLabel` and `description`) so we don't have to pass `{ ...tileProps }` which can get unwieldy quickly (as it's quite opaque as to what props it contains).

Reply:
Yeah, I tried that in https://github.com/Automattic/wp-calypso/pull/22247/commits/53e55481eb4f88f27c7892263ca9345f7ad07ead and you're right that it looks better.